src/lazymerge.jl

```diff
@@ -33,9 +33,7 @@ have one (possibly nested) named tuple for observed data, and another for a
 proposal in an MCMC algorithm. The merge is therefore in the body of a loop
 that's executed many times.
 """
-function lazymerge(x, y)
-    return LazyMerge(x,y)
-end
+function lazymerge end
```

Comment:
```suggestion
lazymerge
```
I think this also works

Reply:
Interesting. Do you know of any docs on this? Any reason to prefer it?

Reply:
https://docs.julialang.org/en/v1/manual/documentation/#Functions-and-Methods-2

Oh actually you are right, your syntax is preferred